HousingSearchApi.Tests/V1/E2ETests/Fixtures/TenureFixture.cs

```diff
@@ -103,5 +103,70 @@ public void GivenSimilarTenures(string paymentReference, string fullAddress, str
 
             Thread.Sleep(10000);
         }
+
+        public void GivenATenureWithSpecificUprn(string uprn)
+        {
+            var fixture = new Fixture();
+            var listOfTenures = new List<QueryableTenure>();
+
+            var tenure = fixture.Create<QueryableTenure>();
+            tenure.TenuredAsset.Uprn = uprn;
+
+            listOfTenures.Add(tenure);
+
+            var awaitable = ElasticSearchClient.IndexManyAsync(listOfTenures, INDEX).ConfigureAwait(true);
+
+            while (!awaitable.GetAwaiter().IsCompleted) { }
+
+            Thread.Sleep(10000);
```

Comment:
Why is this delay required here?

Reply:
Is this perhaps a symptom of other steps being async and this one isn't? Would making the fixture async resolve the issue?

Reply:
I tried making the steps async and using await but that did not work. It seems the BDD tests were built from the ground up to be synchronous as such making them async may require large tweaks to the overall BDD test setup. It might be worth adding it to the backlog as a tech debt ticket as that would be better off being done in a another PR.